### DISCUSSION TITLE
Daily 30/11/2023

### DISCUSSION THREAD

**Discussion by JasonLAGOUTE** [Sinan-Karakaya/R-Type#26] [Daily]
30/11/2023

**Comment by jovanHillion** [comment]
Hello ajd je vais travailler sur l'ia donc pas de R-type pour moi sauf en soirée ou je vais tj plus me pencher sur le projet

**Comment by JasonLAGOUTE** [comment]
Hello ! Je vais pas avoir le temps de taff sur le R-TYPE, surment continuer de me renseigner et bien regarder la lib Asio.

**Comment by lucasdpt** [comment]
Aujourd'hui, mise en place de la base de la lib network

**Comment by Sinan-Karakaya** [comment]
Quelque petits fix de CI de fait ajd
Ce soir je vais normalement pouvoir commencer à taffer sur l'intégration ECS + Serializer
Et ça va taffer la doc en même temps pour que tout le monde puisse se retrouver dans le projet
